src/streamlink/plugins/youtube.py

```diff
@@ -75,7 +76,15 @@ def parse_fmt_list(formatsmap):
                 )
             }]
         ),
-        validate.optional("hlsvp"): validate.text,
+        validate.optional("player_response"): validate.all(
+            validate.text,
+            validate.transform(lambda t: json.loads(t)),
+            {
+                "streamingData" : validate.any ({
```

Comment:
you need `validate.optional("streamingData"): {`

`Unable to validate key 'player_response': Key 'streamingData' not found in`

it will error for +18 videos